+ echo Logging output to experiments/logs/faster_rcnn_alt_opt_VGG16_.txt.2016-09-09_05-57-42
Logging output to experiments/logs/faster_rcnn_alt_opt_VGG16_.txt.2016-09-09_05-57-42
+ ./tools/train_faster_rcnn_alt_opt.py --gpu 1 --net_name VGG16 --weights data/imagenet_models/VGG16.v2.caffemodel --imdb widerface_2016_train --cfg experiments/cfgs/faster_rcnn_alt_opt.yml
Traceback (most recent call last):
  File "./tools/train_faster_rcnn_alt_opt.py", line 19, in <module>
    from datasets.factory import get_imdb
  File "/disk2/workspace/github/ataraxia/faceX/py-faster-rcnn-zyf/tools/../lib/datasets/factory.py", line 14, in <module>
    from datasets.widerface import widerface
  File "/disk2/workspace/github/ataraxia/faceX/py-faster-rcnn-zyf/tools/../lib/datasets/widerface.py", line 205
    ix_cnt = 0
    ^
IndentationError: unexpected indent
